wordpress-mobile/WordPress-iOS
# Feature Introduction: initial view

Ref: #18176

This adds an initial Feature Introduction view that will be used to display information about a feature. Right now, it just shows a `CollapsableHeaderViewController` with no content. 

As noted in [this](pbArwn-44k#comment-5250) discussion, some changes will needed in `CollapsableHeaderViewController` to support Feature Introduction:
- An optional image to the header view. For example, the Blogging Prompts light bulb
- Reconfigure the action buttons.

Because we don't have any official way of showing the feature introduction just yet, I have set it to show in `applicationDidBecomeActive` to facilitate easily accessing it (by minimizing and relaunching the app). This is temporary and will be removed before merging. 

To test:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction should appear with:
  - The header title.
  - The header subtitle.
  - One unconfigured action button.
- The only thing that does work is the close button, so tapping it should dismiss the view. 🎉 

<kbd>![initial_fi](https://user-images.githubusercontent.com/1816888/160024943-1f8bac55-e95e-4f79-96e7-0d335820691b.png)</kbd>

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
